plugin/environment.go

```diff
@@ -250,21 +269,32 @@ func (env *Environment) Activate(id string) (manifest *model.Manifest, activated
 	return pluginInfo.Manifest, true, nil
 }
 
+func (env *Environment) RemovePlugin(id string) bool {
```

Comment:
Same here. Should this be
```suggestion
func (env *Environment) RemovePlugin(id string) error {
```
?

Reply:
Same comment as above, where if the plugin is not in the map, then that just means it's disabled. If it is not in the map, it is not an error. The resolutions mentioned above also apply here.